recipes/hs-process/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "hs-process" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/bb/12/820c4894f45ebeb6338e9fa8c6f070aa5b869b97c4a1fefb04bd70f8417c/hs_process-0.0.3.tar.gz"
+  sha256: ec8adc1562e5d7175341f3a58a0e08fd3f72915fa6a2eba12c841aaf5397cf65
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python ==3.7
```

Comment:
Does this really only work with python 3.7?

Reply:
Python 3.7 is all I've worked on. I do know it does not work on python 3.8, I think due to the goepandas/GDAL dependencies. I'm guessing it will work on other version of python 3 - can this be determined by just changing the meta.yaml file?

Reply:
Just installed and tested with python 2.7, 3.5, 3.6, and 3.7
2.7: passes 16/19 tests (issues with copying a list "'list' object has no attribute 'copy'")
3.5: passes 18/19 tests (issues with converting Series to numpy "'Series' object has no attribute 'to_numpy'")
3.6: passes 19/19 tests

Note that the tests are pretty basic at this point.
3.7: passes 19/19 tests